lib/source-info.coffee

```diff
@@ -37,6 +38,27 @@ module.exports =
         else
           null
 
+    minitestRegExp: (text)->
+      @_minitestRegExp ||= unless @_minitestRegExp 
+        value = text.match(/"(.*?)"/) if text?
+        if value
+          value[1]
+        else
+          ""
+
+    isMiniTest: ->
+      editor = atom.workspace.getActiveTextEditor()
+      cursor = editor and editor.getCursor()
```

Comment:
Due to changing API, this should now be: `editor.getLastCursor()`.

This repeats much of what is on line 34. Can you reuse that logic?